### ISSUE TITLE
Panic when several users are connected with simulcast enabled

### ISSUE THREAD

**Issue by bekriebel** [livekit/livekit-server#102]
**Describe the bug**
The livekit server throws a panic error and stops functioning after several users are connected with simulcast enabled. The error thrown is:
```json
{
  "level": "error",
  "ts": 1630353563.8694067,
  "logger": "livekit",
  "caller": "runtime/panic.go:965",
  "msg": "recovered panic",
  "panic": "runtime error: invalid memory address or nil pointer dereference",
  "error": "runtime error: invalid memory address or nil pointer dereference",
  "stacktrace": "runtime.gopanic\n\t/usr/local/go/src/runtime/panic.go:965\nruntime.panicmem\n\t/usr/local/go/src/runtime/panic.go:212\nruntime.sigpanic\n\t/usr/local/go/src/runtime/signal_unix.go:734\ngithub.com/pion/ion-sfu/pkg/buffer.(*Buffer).GetSenderReportData\n\t/go/pkg/mod/github.com/livekit/ion-sfu@v1.20.8/pkg/buffer/buffer.go:556\ngithub.com/pion/ion-sfu/pkg/sfu.(*WebRTCReceiver).GetSenderReportTime\n\t/go/pkg/mod/github.com/livekit/ion-sfu@v1.20.8/pkg/sfu/receiver.go:414\ngithub.com/pion/ion-sfu/pkg/sfu.(*DownTrack).CreateSenderReport\n\t/go/pkg/mod/github.com/livekit/ion-sfu@v1.20.8/pkg/sfu/downtrack.go:338\ngithub.com/livekit/livekit-server/pkg/rtc.(*ParticipantImpl).downTracksRTCPWorker\n\t/workspace/pkg/rtc/participant.go:912"
}
```

Once this error is thrown, I will see that some video streams stop getting shared to the newest connected user. Shortly after this, signaling events stop being passed between connected users and no new users can connect or reconnect to the server.

**Server**
- Version: master branch at commit 6a88bcc
- Environment: Fly.io using a docker container

**Client**
- SDK: client-sdk-js
- Version: main branch at commit f22916c

**To Reproduce**
The crash is inconsistent, but seems to occur after 6 or 7 users connect to a session, all sharing video and audio with simlcast enabled. If simulcast is not enabled, I have not reproduced the error.

**Expected behavior**
The the error to not be spawned, or for the server to recover cleanly.

**Additional context**
I'm not sure if this affects the issue, but my video streams are all opened with low resolution streams:
```ts
          resolution: {
            width: { ideal: 320 },
            height: { ideal: 240 },
          }
```
Even though I'm not requesting high quality streams, simulcast still seems to be helpful for slower clients.

**Comment by bekriebel** [comment]
The resolution does seem to have something to do with it. I tried with the `client-sdk-js` sample app wit the defaults and it did not seem to spawn the error. However, if I change lines: https://github.com/livekit/client-sdk-js/blob/f22916c215949acc91a11c147fc6394a0165df1d/example/sample.ts#L183-L185 to be:
```ts
      video: {
        deviceId: { ideal: 'default' },
        resolution: {
          width: { ideal: 320 },
          height: { ideal: 240 },
        },
```
I am able to spawn the error within 4-6 users connecting with Edge (chromium) browsers.

**Comment by frostbyte73** [comment]
This should be fixed now in `v0.12.4`

**Comment by bekriebel** [comment]
Confirmed the fix. Thanks!
